### ISSUE TITLE
Error doing job: Task exception was never retrieved 

### ISSUE THREAD

**Issue by mstefany** [rgc99/irrigation_unlimited#38]
Hi,

my Irrigation Unlimited suddenly broke - I cannot enable/disable zones or master.

Only following is visible in the logs:
```
Logger: homeassistant
Source: custom_components/irrigation_unlimited/irrigation_unlimited.py:1771
Integration: Irrigation Unlimited (documentation, issues)
First occurred: 3. září 2021 20:39:16 (317 occurrences)
Last logged: 0:22:01

Error doing job: Task exception was never retrieved
Traceback (most recent call last):
  File "/config/custom_components/irrigation_unlimited/irrigation_unlimited.py", line 2535, in _async_timer
    self.timer(time)
  File "/config/custom_components/irrigation_unlimited/irrigation_unlimited.py", line 2525, in timer
    self.poll_main(time)
  File "/config/custom_components/irrigation_unlimited/irrigation_unlimited.py", line 2519, in poll_main
    self.poll(time, force)
  File "/config/custom_components/irrigation_unlimited/irrigation_unlimited.py", line 2508, in poll
    if self.muster(wtime, force) != 0:
  File "/config/custom_components/irrigation_unlimited/irrigation_unlimited.py", line 2473, in muster
    status |= controller.muster(time, force)
  File "/config/custom_components/irrigation_unlimited/irrigation_unlimited.py", line 1795, in muster
    sequence_status = self.muster_sequence(
  File "/config/custom_components/irrigation_unlimited/irrigation_unlimited.py", line 1771, in muster_sequence
    next_run += duration_max
TypeError: unsupported operand type(s) for +=: 'NoneType' and 'datetime.timedelta'
```

Home Assistant OS 2021.8.8
Raspberry Pi 4B

**Comment by rgc99** [comment]
This happens if you disable the 1st zone in a sequence. Fixed in the latest release 2021.9.0. Please update.

**Comment by mstefany** [comment]
Yes, that seems to be the issue and 2021.9.0 solves it. Thank you!
